Makefile

```diff
@@ -11,7 +11,6 @@ DATA_PROTO_DIR=internal
 # Needed to be able to build amd64 binaries on MacOS M1
 DOCKER_PLATFORM="linux/amd64"
```

Comment:
We should remove this as well and make `make run` work on MacOS as well.

Reply:
`make run` now fails at this step:

/build/install_build_deps.sh because it is trying to download a amd64 package.

```
#14 43.60 + FDB_VERSION=6.3.23
#14 43.60 + [[  == darwin* ]]
#14 43.62 /build/install_build_deps.sh: 28: [[: not found
#14 43.62 + mktemp -p /tmp/ -u
#14 43.63 + FDB_PACKAGE_PATH=/tmp/tmp.bnGXMebWcp-foundationdb-clients_6.3.23-1_amd64.deb
...
#14 46.30 dpkg: error processing archive /tmp/tmp.bnGXMebWcp-foundationdb-clients_6.3.23-1_amd64.deb (--install):
#14 46.30  package architecture (amd64) does not match system (arm64)
```